resources/views/emails/membershipenddate.blade.php

```diff
@@ -0,0 +1,29 @@
+@extends('emails.template')
+
+@section('body')
+
+<p>
+    Dear {{ $user->name }},
+</p>
+
+<p>
+    I am writing to you to let you know that the secretary has set an end date for your membership at S.A. Proto.
+</p>
+<p>
+    Your membership will end at: <b>{{Carbon::createFromTimestamp($user->member->member_untill)->format('d-m-Y')}}.</b><br>
+    This means you are a full member before this date, but will not be for the following academic year.
```

Comment:
I think this should be phrased differently as the end date might be different than September so saying that the membership is no longer valid in the following academic year might be confusing. Maybe change it to: "This means that your membership is valid up until this date and will then be terminated."